### ISSUE TITLE
Create an automated tool to identify broken links in documentation

### ISSUE THREAD

**Issue by mbghsource** [oVirt/ovirt-site#589]
### Description of the problem
There are quite a few broken links in ovirt documentation pages.
Perhaps this is due to transitioning from one wiki system to another.
Whatever the reasons, it caused me a bit of frustration. Many others are likely to be affected by this, risking bad reputation for oVirt.

An automated process, 
- that identifies broken links and marks them prominently on the page as (BROKEN_LINK)
- creates an issue and notifies all previous editors of the page
would help improve the quality of documentation for oVirt.

I would like to help create such a tool, if I can get some guidance.
Thank You.

**Comment by mureinik** [comment]
@maorlipchuk and @danielerez started work on such a tool back in the day, although I'm not sure about its current status:
https://github.com/maorlipchuk/Wiki-Link-Validator

Maor, Daniel - can you show @mbghsource the ropes?

**Comment by mbghsource** [comment]
@mureinik - Thank you for responding promptly.

After noticing that better minds have already worked on this issue, I did what I could've done before filing this issue. Search for tools ;-)

I found a few chrome extensions, that highlighted bad links prominently, and help avoid the pain of discovering them manually. 

Here is a screen recording from trying one such tool (LinkMiner).
https://drive.google.com/file/d/0B9LV8Sb_qIEUNmI3cnFCeENGU28/view?usp=sharing

In the short term, to improve user experience, it might help to post a caution on ovirt documentation site about broken links and provide links to some browser tools  chrome ( https://chrome.google.com/webstore/search/broken%20links ),   firefox ( https://addons.mozilla.org/en-US/firefox/search/?q=broken+links ).
